reactioncommerce/reaction
# add translated text for adding user to group by admin

Resolves #4530
Impact: **minor**  
Type: **bugfix**

## Issue
When an admin add a member to group a wrong message appears in the toast. The message is only the key which needs to be added with associated translations in  **i18n folder**.
You can find the opened issue [4530](https://github.com/reactioncommerce/reaction/issues/4530).

## Solution
Added translated text for **admin.groups.addUserSuccess** in most of the languages

## Testing
1. Log in as an admin.
2. Add user to a group.
3. Check the message appear in green at the left bottom.
